src/test/regress/sql/bfv_partition_plans.sql

```diff
@@ -1,6 +1,38 @@
 create schema bfv_partition_plans;
 set search_path=bfv_partition_plans;
 
```

Comment:
I would suggest to add
`SET client_min_messages = WARNING;`
to avoid excessive **NOTICE** logs for `create table t1` and `create table t2` in *.out files.

Reply:
done